soyuz_app/views/user.py

```diff
@@ -30,6 +38,19 @@ def signup(request, batch_number, user_hubspot_id):
             user.set_password(raw_password)
             user.save()
             batch.users.add(user)
+
+            properties = {"bootcamp_funnel_status": "basics_apply;basics_register"}
+
+            simple_public_object_input = SimplePublicObjectInput(properties=properties)
+            try:
+                api_response = client.crm.contacts.basic_api.update(
+                    contact_id=user_hubspot_id,
+                    simple_public_object_input=simple_public_object_input,
+                )
+                pprint(api_response)
+            except ApiException as e:
+                print("Exception when calling basic_api->update: %s\n" % e)
```

Comment:
these should trigger sentry captures?

I think it needs something like this: https://docs.sentry.io/platforms/python/guides/django/usage/#capturing-errors

```
from sentry_sdk import capture_exception

try:
    a_potentially_failing_function()
except Exception as e:
    # Alternatively the argument can be omitted
    capture_exception(e)
```